src/game/default/g_item.c

```diff
@@ -25,6 +25,17 @@
 const vec3_t ITEM_MINS = { -16.0, -16.0, -16.0 };
 const vec3_t ITEM_MAXS = { 16.0, 16.0, 16.0 };
 
+g_armor_info_t armor_info[] = {
+	{ 100, 200, 0.8, 0.6, ARMOR_BODY },
+	{ 50, 100, 0.6, 0.3, ARMOR_COMBAT },
+	{ 25, 50, 0.3, 0.0, ARMOR_JACKET }
+};
+
+// move this somewhere
+//armor_info[0] = { 100, 200, 0.8, 0.6, ARMOR_BODY };
+//armor_info[1] = { 50, 100, 0.6, 0.3, ARMOR_COMBAT };
+//armor_info[2] = { 25, 50, 0.3, 0.0, ARMOR_JACKET };
+
```

Comment:
Cool. You know what? These could actually be moved right into G_ArmorInfo as `static const`. It seems like they're only ever accessed via that function. I'll take a crack at this.